docs/installation.md

```diff
@@ -26,101 +31,92 @@ mv nextflow ~/bin
 
-This can be also be done on the command line:
+Whilst most parameters can be specified on the command line, it is usually sensible to create a configuration file for your environment.
 
-```
--e.process={executor='SLURM'}
-```
+If you are the only person to be running this pipeline, you can create your config file as `~/.nextflow/config` and it will be applied every time you run Nextflow. Alternatively, save the file anywhere and reference it when running the pipeline with `-c path/to/config`.
 
-Many different cluster types are supported by Nextflow. For more information, please see the [Nextflow documentation](https://www.nextflow.io/docs/latest/executor.html).
+If you think that there are other people using the pipeline who would benefit from your configuration (eg. other common cluster setups), please let us know. We can add a new configuration and profile which can used by specifying `-profile <name>` when running the pipeline.
 
-Note that you may need to specify cluster options, such as a project or queue. To do so, use the `clusterOptions` config option.
+The pipeline comes with several such config profiles - see the installation appendices and usage documentation for more information.
 
-```groovy
-process {
-  executor = 'SLURM'
-  clusterOptions = '-A myproject'
-}
-```
+## 4) Reference Genomes
+The nf-core/methylseq pipeline needs a reference genome for read alignment. Support for many common genomes is built in if running on UPPMAX or AWS, by using [AWS-iGenomes](https://ewels.github.io/AWS-iGenomes/).
 
-### Reference Genomes
-The nf-core/methylseq pipeline needs a reference genome for read alignment. Support for many common genomes is built in if running on UPPMAX or AWS, by using [illumina iGenomes](https://support.illumina.com/sequencing/sequencing_software/igenome.html).
-
-If you don't want to use the illumina iGenomes you can supply either a Bismark reference or a FASTA file. If a Bismark reference is specified, the pipeline won't have to generate it and will be finished quite a bit faster. If a FASTA file is supplied then the Bismark reference will be built when the pipeline starts. Use the command line option `--saveReference` to keep the generated references so that they can be added to your config and used again in the future. Use  `--bismark_index` or `--fasta` to specify the paths to the reference.
-
-Alternatively, you can add the paths to your NextFlow config under a relevant id and just specify this id with `--genome ID` when you run the pipeline:
+Alternatively, ou can supply either a Bismark / BWA reference or a FASTA file. This can be done on the command line (see the [usage docs](usage.md#supplying-reference-indices)).
+You can use the same pipeline config framework as used with iGenomes to specify multiple referenes of your own. Add the paths to your NextFlow config under a relevant id and just specify this id with `--genome ID` when you run the pipeline:
 
 ```groovy
```

Comment:
I'm guessing there's no Nextflow in Markdown yet?

Reply:
There is - it uses the same linguist (?) parsing that the rest of the site uses.

Reply:
Can we specify ```nextflow then?

Reply:
Yup!